app/assets/javascripts/training/components/training_slide_handler.jsx

```diff
@@ -97,10 +110,15 @@ const TrainingSlideHandler = () => {
   // fetches the initial data and sets the base title
   useEffect(() => {
     setBaseTitle(document.title);
+    setNavBreadcrumb(document.getElementById('react_root').getAttribute('data-breadcrumbs'));
```

Comment:
This code has been repeated 3 times. Maybe turn this into a function call?

Reply:
okay done, thanks